README.md

```diff
@@ -151,6 +151,7 @@ Device notes are sorted alphabetically, major (upper case) notes before minor
 | Fan/LED controller | [NZXT RGB & Fan Controller (3+6 channels)](docs/nzxt-hue2-guide.md) | <sup>_hp_</sup> | 1.12.1 |
 | Fan/LED controller | [NZXT Smart Device](docs/nzxt-smart-device-v1-guide.md) | <sup>_h_</sup> | 1.11.1 |
 | Fan/LED controller | [NZXT Smart Device V2](docs/nzxt-hue2-guide.md) | <sup>_h_</sup> | 1.11.1 |
+| FAN/LED controller | [NZXT 2023 RGB Controller](docs/nzxt-smart-device-v1-guide.md) | <sup>_p_</sup> | git |
```

Comment:
```suggestion
| LED controller | [NZXT 2023 RGB Controller](docs/nzxt-hue2-guide.md) | <sup>_p_</sup> | git |
```

further down with the other LED-only controller.

On that note, Isn't this new controller closer to the HUE 2 family? It supports HUE 2 accessories, which the V1 family doesn't. Thus, I think it would be a better fit for `nzxt-hue2-guide.md`.

Or maybe it should get its own driver module and guide?

Finally, why the `p` (partial support) note?